### DISCUSSION TITLE
Is there a way specify a request threshold for a specific item that overrides the station threshold?

### DISCUSSION THREAD

**Discussion by npuldon** [Sharparam/cybersyn-combinator#37] [Q&A]
### Description

Is there a way specify a request threshold for a specific item that overrides the station threshold? I can do this with 'base' cybersyn but I haven't figure out how to do it when this mod is in place.

Sorry if this is not the right place for this post.  You didn't have a category for 'general support/usage' and I'm not sure if this is already a feature or not.

### Steps to reproduce

See above

### Mod version

0.6.1

### Factorio version

1.1.109

### Operating system

Windows 10

**Comment by Sharparam** [comment]
I moved the issue into discussions since I think that's a better fit for general questions like this.

The request threshold overrides are set by inputting signals to the separate Cybersyn combinator set to the station control mode. I'm not sure if there would be an easy/intuitive way to support that directly from this mod, essentially you just put a new constant combinator connected to the station control combinator, and set your override thresholds as signals in it.

(Unless I'm misremembering stuff about how overriding thresholds work, it's been a while since I played Factorio and I never made use of that feature in Cybersyn.)

So it would mean that this mod would need to connect to both of the Cybersyn combinators, and know where to send the signals. Maybe that's possible if embedding another "hidden" constant combinator entity in it? (Kinda leaving this as an open question and also as a note for myself for the future.)
